setup.py

```diff
@@ -25,10 +26,23 @@
     install_requires.append('idna')
     install_requires.append('requests[security]')
 
+def read(rel_path):
+    here = os.path.abspath(os.path.dirname(__file__))
+    with open(os.path.join(here, rel_path), 'r') as fp:
+        return fp.read()
+
+def get_version(rel_path):
+    for line in read(rel_path).splitlines():
+        if line.startswith('__version__'):
+            delim = '"' if '"' in line else "'"
+            return line.split(delim)[1]
+    else:
+        raise RuntimeError("Unable to find a valid __version__ string in %s." % rel_path)
```

Comment:
Is this a cool python trick to put else: after a for loop or was the tab indentation just off here? I suppose we could just have it be `raise` without the `else:` to the same effect?

Reply:
Yes, `for/else` is a pythonic way. `else` statement is triggered when the loop ends until the end without any interruption (break/return). More like a `then` stage for the loop. 

Yes, it's the same as just keeping the `raise` there as you said, I'll update it.